packages/schematics/angular/ng-new/schema.json

```diff
@@ -132,12 +132,12 @@
       "alias": "S"
     },
     "createApplication": {
-      "description": "When true (the default), creates a new initial app project in the new workspace.",
+      "description": "When true (the default), creates a new initial app project in the src folder of the new workspace. When false, creates an empty workspace with no initial app. You can then use the generate application command so that all apps are created in the projects folder.",
       "type": "boolean",
       "default": true
     },
     "minimal": {
-      "description": "When true, creates a project without any testing frameworks and should be used for learning purposes only.",
+      "description": "When true, creates a project without any testing frameworks. (Use for learning purposes only).",
```

Comment:
Can we also update the minimal description in the other schemes. So that they are all aligned?

See https://github.com/angular/angular-cli/pull/13408/files for reference.